config/migrations/besluit-document-type-concept-scheme.sparql

```diff
@@ -0,0 +1,56 @@
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/application> {
```

Comment:
Every migration in this PR will add their information to the `http://mu.semte.ch/application` graph